Open source - Resolves 336 (#518)
Codecademy/docs: content/general/concepts/open-source/open-source.md

---
Title: 'Open Source'

> Description: '"Open source" refers to software that is released with the source code made freely available for modification and possible redistribution.'

Reviewer: ```suggestion
Description: 'Open source refers to software that is released with the source code made freely available for modification and possible redistribution.'
```

I don't think we need quotes in the meta-`Description`.

Subjects:

>   - 'Computer Science'

Reviewer: ```suggestion
  - 'Code Foundations'
  - 'Computer Science'
```

Tags:
  - 'Conceptual'
  - 'Development'
  - 'Linux'
CatalogContent:
  - 'paths/code-foundations'
  - 'paths/computer-science'
---

> "Open source" refers to software that is released with the source code made freely available for modification and possible redistribution. Open source projects allow the code to be reused by anyone with the restriction that any derivative projects are released under the same license. Such projects are intended to foster open collaboration, with many developers reviewing and updating the code. One benefit of this is that it provides greater opportunity for bugs and security holes to be detected and fixed. Open source software is also thought to be more trustworthy, since all the code is available to the public, any unwanted or nefarious code will be quickly exposed by the development community surrounding the project.

Reviewer: ```suggestion
The term "open source" refers to softwares that are freely available for modification and possible redistribution. Open source projects allow the source code to be reused by anyone with the restriction that any derivative projects are released under the same license. 

Such projects are intended to foster open collaboration from a variety of perspectives on problem-solving. The code is reviewed and updated by groups of developers of all sizes. One benefit of this is that it provides greater opportunity for bugs and security holes to be detected and handled effectively. 

Open source software (OSS) is also thought to be more trustworthy. Since all the code is available to the public, any unwanted or nefarious behavior or activity will be quickly exposed by the development community surrounding the project.
```

> ## Examples of Open Source Software

Reviewer: ```suggestion
## Examples of OSS
```

---
Title: 'Open Source'
Description: 'Open source refers to software that is released with the source code made freely available for modification and possible redistribution.'
Subjects:
  - 'Code Foundations'
  - 'Computer Science'
Tags:
  - 'Conceptual'
  - 'Development'
  - 'Linux'
CatalogContent:
  - 'paths/code-foundations'
  - 'paths/computer-science'
---

> The term "open source" refers to softwares that are freely available for modification and possible redistribution. Open source projects allow the source code to be reused by anyone with the restriction that any derivative projects are released under the same license.

Reviewer 2: I had to look this up but apparently "softwares" is not a word and the plural of software is software. I have been enlightened! 🙏

```suggestion
The term "open source" refers to software that is freely available for modification and possible redistribution. Open source projects allow the source code to be reused by anyone with the restriction that any derivative projects are released under the same license.
```

> Such projects are intended to foster open collaboration from a variety of perspectives on problem-solving. The code is reviewed and updated by groups of developers of all sizes. One benefit of this is that it provides greater opportunity for bugs and security holes to be detected and handled effectively.

Reviewer 2: ```suggestion
Such projects are intended to foster open collaboration from a variety of perspectives on problem-solving. The code is reviewed and updated by groups of developers of all sizes. One benefit of this is that it provides a greater opportunity for bugs and security holes to be detected and handled effectively.
```

Open source software (OSS) is also thought to be more trustworthy. Since all the code is available to the public, any unwanted or nefarious behavior or activity will be quickly exposed by the development community surrounding the project.

## Examples of OSS

> - Linux: Ever since the Linux kernel was released by [Linus Torvalds](https://www.codecademy.com/resources/docs/general/linus-torvalds) in 1991, an endless variety of Linux distributions have been produced, all similarly open source. The majority of software for the Linux operating system is open source as well.

Reviewer 3: ```suggestion
- [Linux](https://www.linux.org/): Ever since the Linux kernel was released by [Linus Torvalds](https://www.codecademy.com/resources/docs/general/linus-torvalds) in 1991, an endless variety of Linux distributions have been produced, all similarly open source. The majority of software for the Linux operating system is open source as well.
```

Reviewer 3: @SSwiniarski Took the liberty of giving this a look-through! Some minor things

- LibreOffice: a competitor to Microsoft Office, that offers word processing, spreadsheet, presentation and database software.

> - MySQL: a database which is commonly used as a back-end database on many websites.

Reviewer 2: ```suggestion
- MySQL: a database that is commonly used as a back-end database on many websites.
```

> - LibreOffice: a competitor to Microsoft Office, which offers word processing, spreadsheet, presentation, and database software.
> - MySQL: a database that is commonly used as a back-end database on many websites.

Reviewer 3: ```suggestion
- [LibreOffice](https://www.libreoffice.org/): A competitor to Microsoft Office, which offers word processing, spreadsheet, presentation, and database software.
- [MySQL](https://www.mysql.com/): A database that is commonly used as a back-end database on many websites.
```
